src/components/Bar.jsx

```diff
@@ -8,7 +8,7 @@ import { shouldEnableTracking, getTracker, configureTracker } from '../lib/piwik
 // import Apps from './Apps'
 // import Settings from './Settings'
 // import Drawer from './Drawer'
-// import Claudy from './Claudy'
+import Claudy from './Claudy'
```

Comment:
At the end, the comments are uncommented in further commits. Hard to review commit by commit this way.